### DISCUSSION TITLE
Rabbit MQ plugin .ez not generated

### DISCUSSION THREAD

**Discussion by NuwanSameera** [rabbitmq/rabbitmq-server#5246] [Other] [answered]
I clone the repository and modify rabbitmq_management plugin by adding new function for my use. All the code changes completed and tested using `make run-broker`. Now I need to generate executable file. I followed [1] for plugin development and try with `make dist`. It is completed without any error. Plugins folder created and there are sub folders for each plugin. But .ez file not available in anywhere. 

What is the mistake.

[1]. https://www.rabbitmq.com/plugin-development.html

**Comment by gomoripeti** [comment, marked as answer]
try the below (based on https://github.com/rabbitmq/rabbitmq-server/blob/master/deps/rabbit_common/mk/rabbitmq-dist.mk#L8)
```
make dist DIST_AS_EZS=true
```
the plugin dev docs needs to be updated

**Reply by NuwanSameera** [reply]
Thank you very much.  Working properly
